app/views/candidate_interface/referees/type.html.erb

```diff
@@ -3,7 +3,7 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_with model: @reference_type_form, url: candidate_interface_update_referees_type_path, method: :post do |f| %>
+    <%= form_with model: @reference_type_form, url: candidate_interface_update_referees_type_path(id: @id), method: :post do |f| %>
       <%= f.govuk_error_summary %>
       <h1 class="govuk-heading-xl">
         <% title = @referees.any? ? 'Second referee' : 'First referee' %>
```

Comment:
Smol bug, when I:

1. Added my first referee
2. Click `Change` for `Reference type`
3. I see `Second referee` as the heading, I should see `First referee`

Maybe the below will solve this.

```suggestion
        <% title = @referees.one? ? 'First referee' : 'Second referee' %>
```

Reply:
Aww, this will cause an issue when you add the first referee, because `@referees.one?` will be false

Reply:
resolved this in 1066d44 along with the issue on the reference page(it was an existing bug it always showed First Referee)